examples/power_distribution.py

```diff
@@ -23,6 +23,7 @@
 from frequenz.sdk._data_ingestion import MicrogridData
 from frequenz.sdk._data_ingestion.formula_calculator import FormulaCalculator
 from frequenz.sdk.actor import actor
+from frequenz.sdk.actor import run as actor_run
```

Comment:
Why are you aliasing this function? If you think users most likely will like to alias it, then I think we should change the function's name. Otherwise using this instead might be clear enough:

```py
from frequenz import actor

actor.run(...)
```

This is actually google's recommended way to import stuff, and I think it makes sense, but sadly we are not doing this consistently in the SDK, so I guess using a more explicit function name is OK too.

In any case, I would call this `run_actors` instead of `actor_run`.

Reply:
I think I've added an alias because the module `examples/power_distribution` has a method `run()` but it is true that your suggestion would work. I'll work out the changes and I kept the import of the `actor` decorator as it is handy to use `@actor`